tools/win_compile_sanity_check.ps1

```diff
@@ -0,0 +1,28 @@
+# Script to test if a small target builds quickly on windows as expected.

```

Comment:
Please rename the b/239087769 for rationale/examples

Reply:
Also, consider putting it under `tools/workflow` or `tools/windows` or such. Toplevel `tools` dir will get crowded fast

Reply:
done